configs/olimex-stm32-e407/uros/defconfig

```diff
@@ -35,71 +54,24 @@ CONFIG_RAW_BINARY=y
 CONFIG_RR_INTERVAL=200
 CONFIG_SCHED_WAITPID=y
 CONFIG_SDCLONE_DISABLE=y
+CONFIG_SERIAL_TERMIOS=y
 CONFIG_START_DAY=6
 CONFIG_START_MONTH=12
 CONFIG_START_YEAR=2011
 CONFIG_STM32_JTAG_SW_ENABLE=y
+CONFIG_STM32_OTGFS=y
 CONFIG_STM32_PWR=y
-CONFIG_STM32_USART2=n
+CONFIG_STM32_USART3=y
 CONFIG_SYSTEM_NSH=y
 CONFIG_SYSTEM_NSH_CXXINITIALIZE=y
-CONFIG_USART2_RXBUFSIZE=128
-CONFIG_USART2_SERIAL_CONSOLE=y
-CONFIG_USART2_TXBUFSIZE=128
-CONFIG_USER_ENTRYPOINT="nsh_main"
-
-# Enable C99 bool type
-CONFIG_C99_BOOL8=y
-
-# Enable uClibc++
 CONFIG_UCLIBCXX=y
-
-# Enable monotonic clock
-CONFIG_CLOCK_MONOTONIC=y
-
-# Enable uROS example
+CONFIG_UROS=y
 CONFIG_UROS_EXAMPLES=y
+CONFIG_UROS_EXAMPLES_KOBUKI=y
+CONFIG_UROS_EXAMPLES_MYAPP=y
```

Comment:
I think that we should split the examples into different defconfig file, because it could be confusing to see a few applications that are not the basic example.